pkg/arvo/app/radar.hoon

```diff
@@ -0,0 +1,304 @@
+/+  *server
+|%
++$  move  [bone card]
++$  card
+  $%  [%poke wire dock pear]
+      [%wait wire @da]
+      [%rest wire @da]
+      [%info wire toro:clay]
+      [%connect wire binding:eyre term]
+      [%http-response =http-event:http]
+  ==
+::
++$  pear
+  $%  [%noun cord]
+      [%helm-hi cord]
+  ==
+::
++$  state
+  $:  scanning=_|                        :: are we currently scanning?
+      filter=(set ship)
+      configured=(set ship)              :: set of ships to scan
+      outstanding=(map ship @da)         :: ships we've |hi'd with no reply
+      log=(map ship (list [@da @da]))    :: log of past |hi's sent/recv by ship
+      timers=(map wire @da)              :: map of delay timers
+      moves=(list move)                  :: list of outgoing moves
+  ==
+::
+--
+|_  [bol=bowl:gall sty=state]
+::
+++  this  .
+::
+++  abet
+  ^-  (quip move _this)
+  [moves.sty this(moves.sty ~)]
```

Comment:
This should `flop` `moves.sty`, else your moves will be emitted in reverse order.  In practice I don't think it matters for this app, but it's important to get this right because if anything changes so that order does matter, it's very easy to trip over it.

See https://github.com/urbit/arvo/pull/1174#issuecomment-493274913 for context, but the general rule is that the list of moves you produce is evaluated from head to tail.  This leads to the rule that "if you're adding moves `=.  moves  [new-move moves]` then you need to flop at the end".  Otherwise, this:

```
=|  moves=(list move)
=.  moves  [move-1 moves]
=.  moves  [move-2 moves]
=.  moves  [move-3 moves]
moves
```

gives

```
~[moves-3 moves-2 moves-1]
```

which is backwards.